src/DependencyInjection/SyliusGtmEnhancedEcommerceExtension.php

```diff
@@ -25,7 +25,7 @@ public function load(array $configs, ContainerBuilder $container)
 
             $container->setParameter($parameter, $setting);
 
-            if ($setting === true) {
+            if ($setting === true || (is_array($setting) && $setting['enabled'] === true)) {
```

Comment:
Can we be 100% sure that `enabled` is inside `$setting` if `$setting` is an array?

Reply:
I thing I didn't check it because it's how this special field is working, symfony will create it anyway.